test/common/index.js

```diff
@@ -460,7 +460,7 @@ function busyLoop(time) {
 
 function isAlive(pid) {
   try {
-    process.kill(pid, 'SIGCONT');
+    process.kill(pid, 0);
```

Comment:
I'm not convinced this is likely to be the cause unless the tests in question are also using `SIGSTOP`. `process.kill` is asynchronous and should only throw if the `pid` (which appears to be the intention for `isAlive()`) or `signal` is invalid.

Reply:
Yeah, I'm kind of flinging this against the wall to see if it works. (And CI came back and it didn't work.)

Reply:
I haven't read the context here, but I can confirm that `0` is the correct signal to use to check for process existence, and it is has specific support for this purpose on Windows: https://github.com/nodejs/node/blob/01c5c16aba178e8cd87cd5dc026e995a014440fb/deps/uv/src/win/process.c#L1219